test/unit/models/events.test.js

```diff
@@ -188,4 +188,26 @@ describe("Events", function () {
       });
     });
   });
+
+  describe("getPeerById", function () {
+    it("should get peer by id", async function () {
+      const docRef = await eventModel.add(eventData);
+
+      const peerData = {
+        peerId: "dummyid",
+        name: "Satyam Bajpai",
+        eventId: docRef.id,
+        role: "guest",
+        joinedAt: new Date(),
+      };
```

Comment:
Move this also

Reply:
actually I am taking creating the event and then taking doc id of event for peer, so I think it's good to keep it here

Reply:
Ok but you can use the fixture that you already created and replace it with the new value of eventId 
as it may help in decrease the code line and reusability